### PR TITLE
Update Test Data Genny to use POST requests

### PR DESCRIPTION
**Pull request by benjstephenson**

### CODE REVIEW

**Comment by henricook** [comment]
@benjstephenson Did you say this is the one causing acceptance test failures that's not ready yet?

**Comment by benjstephenson** [comment]
@henricook yep, but it's all resolved now!

**Comment by henricook** [comment]
I'll review after today's release is tagged

**Comment by benjstephenson** [comment]
@henricook  grand, jsut running local tests and then I'll remove the DNM tag

**Comment by henricook** [comment]
@benjstephenson Some conflicts for you, then i can merge

File: app/model/command/testdata/# [hunk 1]
```diff
@@ -14,12 +14,62 @@
...
  */
 
-package services.testdata
+package model.command.testdata
 
 import model.ApplicationRoute
 import model.EvaluationResults.Result
-import model.command.testdata.CreateCandidateInStatusRequest$
-import org.joda.time.{DateTime, LocalDate}
+import model.exchange.testdata._
```
Comment (henricook) [inline app/model/command/testdata/#]: @benjstephenson Did you mean to move this into a file called '#'?

**Comment by henricook** [comment]
Tests pass locally, ignoring Bad guy travis

**Pull request merged**